show/bgp_frr_v4.py

```diff
@@ -45,3 +45,34 @@ def neighbors(ipaddress, info_type):
     command += '"'
 
     run_command(command)
+
+# 'network' subcommand ("show ip bgp network")
+@bgp.command()
+@click.argument('ipaddress', metavar='[IPADDRESS|PREFIX]', required=False)
```

Comment:
Suggest to change metavar to match help text in Command Reference (I prefer the case/phrasing). I.e., `metavar='[<ipv4-address>|<ipv4-prefix>]`

Reply:
fixed.